app/assets/stylesheets/application.css.scss

```diff
@@ -18,3 +18,170 @@
 @import "bootstrap";
 @import "font-awesome-sprockets";
 @import "font-awesome";
+
+body {
+  background-color: #ebe7df;
+  color: #333333;
+  font-family: "century gothic", "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-size: 14px;
+  line-height: 1.42857143;
+}
+
+.row h1 {
+  background: #418a95;
+  color: #fff;
+  font-weight: bold;
+  font-size: 20;
+  margin-bottom: 0;
+  padding: 30 10;
+}
+
+.table {
+  box-shadow: 0 10 10 rgba(0, 0, 0, 0.2);
+  margin-bottom: 20;
+  max-width: 100%;
+  width: 100%;
+}
+
+.table tr {
+  border-top: 1px solid #ebe7df;
+}
+
+.table thead tr {
+  border-bottom: 1px solid #ebe7df;
+  border-top: 1px solid #63acb7;
+  background: #63acb7;
+  color: #fff;
+}
+
+.table thead tr:first-child {
+  border-top: none
+}
+
+.table tbody {
+  background: #ccc;
+}
+
+tr:nth-child(even) td {
+  background: linear-gradient(to bottom, #f2f1e8 0%, #f6f7ea 100%);
+}
+
+
+/* Cells in odd rows (1,3,5...) are another (excludes header cells)  */
+
+tr:nth-child(odd) td {
+  background: #FEFEFE;
+}
+
+.table>thead>tr>th,
+.table>thead>tr>td,
+.table>tbody>tr>th,
+.table>tbody>tr>td,
+.table>tfoot>tr>th,
+.table>tfoot>tr>td {
+  line-height: 1.42857143;
+  padding: 10;
+  vertical-align: top;
+}
+
+.table>thead>tr>th {
+  border-bottom: 2px solid #ddd;
+  text-transform: uppercase;
+  vertical-align: bottom;
+}
+
+.btn-primary {
+  color: #fff;
+  background-color: #535353;
+  border-color: #ccc;
+  border-radius: 0;
+}
+
+.btn-primary:hover {
+  color: #fff;
+  background-color: #000;
+  border-color: #000;
+}
+
+.btn-primary:hover {
+  color: #fff;
+  background-color: #000;
+  border-color: #000;
+}
+
+.navbar-brand {
+  float: left;
+  font-family: "code light", "century gothic", Arial, Helvetica, sans-serif;
+  font-size: 28px;
+  font-weight: bold;
+  height: 50;
+  line-height: 20;
+  padding: 15px;
+  text-transform: uppercase;
+}
+
+.navbar-brand span {
+  font-weight: normal;
+}
+
+.navbar-brand:hover,
+.navbar-brand:focus {
+  text-decoration: none;
+}
+
+.container a.navbar-brand:hover,
+.container a.navbar-brand:focus {
+  color: #fff;
+}
+
+.navbar-default {
+  background-color: #535353;
+  border-color: #ccc;
+  box-shadow: 1px 1px 10 #000;
+}
+
+.navbar-default .navbar-brand {
+  color: #fff
+}
+
+.navbar-default .navbar-nav>li>a {
```

Comment:
Merge rule `.navbar-default .navbar-nav>li>a` with rule on line 137<br>Selector should have depth of applicability no greater than 2, but was 4